### DISCUSSION TITLE
"Shared" folders missing in the latest update Mountain Duck 4.17.0

### DISCUSSION THREAD

**Discussion by harisillahi** [iterate-ch/cyberduck#16631] [Q&A] [answered]
I recently updated to the latest version of Mountain Duck and noticed that the "Shared with Me" folders from my friends and colleagues are no longer visible. Previously, I could easily access these shared folders, but now I only see my own files.

This issue is happening with both my Google Drive and OneDrive connections. I'm using a Mac, and I've tried checking for additional updates and reauthorizing my account, but the problem persists.

Has anyone else experienced this issue? If so, do you have any suggestions on how to resolve it?

I've attached an image to show how the Google Drive connection looked in the previous version for reference.

![image](https://github.com/user-attachments/assets/de25d4a5-89d2-4747-9708-4b0e87e8904c)

**Comment by harisillahi** [comment]
I've found that the "Shared with Me" folders are visible again when I roll back to version 4.16.3 of Mountain Duck. This seems to resolve the issue temporarily, but a more permanent fix would be appreciated.

<img width="544" alt="image" src="https://github.com/user-attachments/assets/93028288-2b2a-4898-ad69-17590bc190de">

**Comment by dkocher** [comment]
Set the Path in the Bookmark to `/` to show the top level folders in Google Drive [^1] and OneDrive [^2]. 

[^1]: https://docs.cyberduck.io/protocols/googledrive/#top-level-folders
[^2]: https://docs.cyberduck.io/protocols/onedrive/#top-level-folders

**Comment by harisillahi** [comment]
@dkocher Thank you for the suggestion.

but In the previous version, the "Shared with Me" folders appeared by default without needing to set the path in the bookmark. When I update to the latest version, 4.17.0, the "Shared with Me" folders are not visible in the directory list.

Could you please advise on how to access the "Shared with Me" folder if it doesn't appear automatically in this latest version?

**Reply by dkocher** [reply, marked as answer]
The change in the latest version is to connect to your own files by default with no custom configuration in _Path_. To revert the behaviour to list other top level virtual folders like _Shared with Me_, enter `/` for the _Path_ in the bookmark configuration.

**Reply by harisillahi** [reply]
Thank you @dkocher for your help. I apologize for the confusion earlier—I didn't fully understand the previous comment. Now that it's clear, everything is working well, and I can access back my shared directory.
